<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticDOClass</class>
 <widget class="QDialog" name="StaticDOClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Static DO</string>
  </property>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>-6</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>191</width>
     <height>91</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/StaticDO/Resources/Background.png)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>69</y>
     <width>369</width>
     <height>317</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>369</width>
     <height>317</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>369</width>
     <height>317</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>369</width>
     <height>317</height>
    </size>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>341</width>
      <height>0</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>341</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>396</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>361</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>361</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Port No.    Bit7                          4            3                             0         Hex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="timerFreqLabel">
   <property name="geometry">
    <rect>
     <x>723</x>
     <y>182</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>41</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>50ms</string>
   </property>
  </widget>
  <widget class="QSlider" name="timerTrackBar">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="FFTChoose">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPointCount">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>113</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>400</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>250</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel select:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interval set:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output set:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
